### DISCUSSION TITLE
Monthly Meetings

### DISCUSSION THREAD

**Discussion by dwalkes** [OE4T/meta-tegra#515] [General]
See comments below for monthly meetings/notes.

See the calendar at https://teamup.com/ksfr2rp3ee3bzhq6q1 if you'd like to import this to your calendar of choice.

For other ways to stay in touch with meta-tegra development, see https://github.com/OE4T/meta-tegra/wiki

**Comment by dwalkes** [comment]
Hi Folks,
Our January Monthly meeting will be held January 14th at 9AM MT, 8AM PT, 16:00 UTC.  See calendar link at https://teamup.com/event/show/id/AczmMYMAwt7mVdabgDxrwyC3g9GhGX for details.  Hope you can attend!

**Comment by dwalkes** [comment]
Hi Everyone,
Our February Monthly meeting will be held Feb 11th at 9AM MT, 8AM PT, 1600 UTC.  See calendar link at https://teamup.com/event/show/id/bTSaWzYD7ARTa8axLr9nrpbtJE5XB5 for details.  Hope you can attend!

**Comment by dwalkes** [comment]
Hi Everyone!
Our March monthly meeting will be held March 11th at 9AM MT, 8AM PT, 1600 UTC. See calendar link at https://teamup.com/event/show/id/b2QMxW8zepQm63uwnDdRwwJwYZComg for details. Hope you can attend!

**Comment by paroque28** [comment]
Hi @dwalkes ,
Do you have notes from past meetings?

**Reply by dwalkes** [reply]
@paroque28 I do have my own notes but I haven't shared these in the past because I haven't told people I would share them or gotten their approval.  I think I should probably at least start sharing summaries going forward.  I'll discuss this in the meeting this week.

**Reply by dwalkes** [reply]
Meeting notes from the most recent meeting today can be found at https://docs.google.com/document/d/e/2PACX-1vShY0B20I5JDb00u1nR8fDbBJMiyXoh-TgY87LCHQCtCK7fvK0jAIVggNoEZhiG01ItSahUDBnv_TLY/pub

**Comment by dwalkes** [comment]
Hi Everyone,
Our next monthly OE4T meeting will be held Thursday April 8th at 9AM MT, 8AM PT.   See calendar link at https://teamup.com/event/show/id/F7QANNSFh1kUWP2RoqXyG4r486vrvv.  Hope you can attend!

**Reply by dwalkes** [reply]
You can find the notes and video at [this link](https://docs.google.com/document/d/e/2PACX-1vQg920EKbo5JayANjtyMkmGPO8uUtLrmdVnukp6NdnLQJrcuuBbQS8pictBOpdnQ4CnYbVK0N36Y99V/pub).

**Comment by dwalkes** [comment]
Hi Everyone,
Our next monthly OE4T meeting will be held Thursday May 13th at 9AM MT, 8AM PT. See calendar link at
https://teamup.com/event/show/id/soGrzoCDQo3XVmBcdAW6RxGyNmN4f9.  Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes are at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes---2021-05-13

**Comment by dwalkes** [comment]
Hi Everyone,
Our June Monthly meeting will be held Thursday June 10th at 9AM MT, 8AM PT. See calendar link at https://teamup.com/event/show/id/46LNB1AeCn35BtzspAq6m8Bwmzhy18 for details

**Reply by dwalkes** [reply]
Notes can be found at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-06-10

**Comment by dwalkes** [comment]
Hi Everyone,
Our July monthly meeting will be July 8th at 9AM MDT.  See calendar link at https://teamup.com/event/show/id/pus6eH33R7WkF32Kjm5DHFoTRF9GLP for details.  Hope you can attend!

**Reply by dwalkes** [reply]
Notes can be found at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-07-08

**Comment by dwalkes** [comment]
Hi Everyone,
Our August monthly meeting will be August 12th at 9AM MDT.  See calendar link at  https://teamup.com/event/show/id/gtzmdQwJHLABcVt7vzUTSahAc8EYWr for details.  Hope you can attend!

**Reply by dwalkes** [reply]
Notes can be found at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-08-12

**Comment by dwalkes** [comment]
Hi Everyone,
Our September monthly meeting will be September 9th at 9AM MDT. 
 See calendar link at https://teamup.com/event/show/id/Rn6HvmfkjcirJ5upoLSs9C4cFX54hQ for details.  Hope you can attend!

**Reply by dwalkes** [reply]
Notes can be found at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-09-09

**Comment by dwalkes** [comment]
Hi Everyone,
Our October monthly meeting will be October 14th at 9AM MDT.
See calendar link at https://teamup.com/event/show/id/jVGqBaAFFUDtqoJ4oAJLL78vzuwvSW for details.  Hope you can attend!
Please note - moving to Zoom for future meetings to resolve issues with recordings.

**Reply by dwalkes** [reply]
See https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-10-14

**Comment by dwalkes** [comment]
HI Everyone,
Our November monthly meeting will be November 11th at 9AM MST (note the daylight savings change).
See calendar link at https://teamup.com/event/show/id/J8kbKrnzt7dLtCWkemt9CwZSq4AYzQ for details.

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-11-11

**Comment by dwalkes** [comment]
Hi Everyone,
Our December monthly meeting will be December 9th at 9AM MST
See calendar link at https://teamup.com/event/show/id/mFb1QX4uvjvputyVpVGp8iT4d2QpHe for details.
Hope you can attend!

**Reply by dwalkes** [reply]
See meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2021-12-09

**Comment by dwalkes** [comment]
Hi Everyone,
Our January monthly meeting will be held January 13th at 9AM MST
See calendar link at https://teamup.com/event/show/id/Rsx2wo2ea54v9f2EgMrXdk2PxxYiob for details.
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-01-13

**Comment by dwalkes** [comment]
Hi Everyone,
Our February monthly meeting will be held February 10th at 9:30 AM MT.  **Please note** the half hour later shift in start time.  See calendar link at https://teamup.com/event/show/id/NHSiqHRNgtcnS2qKZqH581zHwe4iH6.  Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-02-10

**Comment by dwalkes** [comment]
Hi Everyone,
Our March monthly meeting will be held March 10th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/bC8NUHjyXHJoHxN5HAPfB681HyqJJj.  Hope you can attend!

**Reply by dwalkes** [reply]
see notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-03-10

**Comment by dwalkes** [comment]
Hi Everyone,
Our April monthly meeting will be held April 14th at 9:30 AM MT.  Please note that the US has switched into Daylight Savings Time since our last meeting.  See calendar link at https://teamup.com/event/show/id/sJaeZCr8FxbzuRBHN9oJxh6rdWCoqM.  Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-04-14

**Comment by dwalkes** [comment]
Hi Everyone,
Our May monthly meeting will be held May 12th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/6TFKYhYXDfNR3Fn8oSXDPZB2dARVve. Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-05-12

**Comment by dwalkes** [comment]
Hi Everyone,
Our June Monthly Meeting will be held June 9th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/XKkZJrDa3HpdEGrCfvphWeu7XFtn7n.  Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-06-09

**Comment by dwalkes** [comment]
Hi Everyone,
Our July Monthly Meeting will be held July 14th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/fznfK7GZpHgn8rn1foZE9eNU6V5n1S.  Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-04-17

**Comment by dwalkes** [comment]
Hi Everyone,
Our August Monthly Meeting will be held August 11th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/5gghZ3iBoADAgLzwteJXKaFLZci5Zq.  Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-08-11

**Comment by dwalkes** [comment]
Hi Everyone,
Our September Monthly Meeting will be held September 8th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/hkmYmnKwxsAchJGqMDG6d4fm3VMxia.  Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-09-08

**Comment by dwalkes** [comment]
Hi Everyone,
Our October Monthly Meeting will be held October 13th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/Jd7dBDNz57GYNdcyirzJkWzTCv5yYT.  Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-10-13

**Comment by dwalkes** [comment]
Hi Everyone,
Our November Monthly Meeting will be held November 10th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/3PjSridJXQeF2wY8rzjMw1BqkMarAZ. Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-11-10

**Comment by dwalkes** [comment]
Hi Everyone,
Our December Monthly Meeting will be held December 8th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/M8hYML3qnoGKo2GoWMUxZuytRoaXLd.  Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2022-12-8

**Comment by dwalkes** [comment]
Hi Everyone,
Our January Monthly Meeting will be held January 12th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/hE8sbhkr6p53gv2fQTFgWB4nyxTRqk.  Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-01-12

**Comment by dwalkes** [comment]
Hi Everyone,
Our February Monthly Meeting will be held Feb 9th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/Gi2BttWMgVHHL3W47AmRjPr2tVTstP

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-02-09

**Comment by dwalkes** [comment]
Hi Everyone,
Our March Monthly Meeting will be held March 9th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/ZbxcrHzXP3YagARxkXguXwthjUzBqg

Hope you can attend!

**Reply by dwalkes** [reply]
See meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-03-09

**Comment by dwalkes** [comment]
Hi Everyone,
Our April Monthly Meeting will be held April 13th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/7mXdPK1iq3FC4RFGBD8cG9rrsvTidH

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-04-13

**Comment by dwalkes** [comment]
Hi Everyone,
Our May Monthly Meeting will be held May 11th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/uTvMMvEEhBCCoRcRbRAHkXUv8DUAUT

Hope you can attend!

**Reply by dwalkes** [reply]
Notes in https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-05-11

**Comment by dwalkes** [comment]
Hi Everyone,
Our June Monthly Meeting will be held June 8th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/5QPFRbcJXsmaK9Kc7vDzGnnQX3TUJD

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-06-08

**Comment by dwalkes** [comment]
Hi Everyone,
Our July Monthly Meeting will be held July 13th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/9neGuV779gKAgvXzc1fjFYnYsEUyuM

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-07-13

**Comment by dwalkes** [comment]
Hi Everyone,
Our August Monthly Meeting will be held August 10th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/PovJzJoUhtzT76r5ycXtLtnRdt83TA

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-08-10

**Comment by dwalkes** [comment]
Hi Everyone,
Our September Monthly Meeting will be held September 14th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/SzxLDXzsaCBxu23hFa8DB3ZRa78K7V

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-09-14

**Comment by dwalkes** [comment]
Hi Everyone,
Our October Monthly Meeting will be held October 12th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/6hPY6t4axkomMJ5S2AwcpoDmMqudcc

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-10-12

**Comment by dwalkes** [comment]
Hi Everyone,
Our November Monthly Meeting will be held November 9th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/4KFnbhb6NnD5avHX78gEaVNzm7HTNE

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-11-09

**Comment by dwalkes** [comment]
Hi Everyone,
Our December Monthly Meeting will be Thursday December 14th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/tHwa6Yhwwem8dwiCGftxFBnjmT67rd

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-12-14

**Comment by dwalkes** [comment]
Hi Everyone,
Our January Monthly meeting will be held Thursday January 11th at 9:30 AM MT.  See calendar link at https://teamup.com/event/show/id/C7CNDGpZy1mP39QkEKtKAYXeeqs9EG

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024-01-11

**Comment by dwalkes** [comment]
Hi Everyone,
Our February Monthly meeting will be held Thursday Feb 8th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/WqqBm1Qbr1XujDAzXsXP5B4Tus3ASf 

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024-02-08

**Comment by dwalkes** [comment]
Hi Everyone,
Our March Monthly meeting will be held Thursday March 14th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/MV1UmpdQqPM68knRrCnZ1kArfYADvZ

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9003%E2%80%9014

**Comment by dwalkes** [comment]
Hi Everyone,
Our April monthly meeting will be held Thursday April 11th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/JFtXqzKJQ7kb1pvkTpqTrwwepAmwzv

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024-04-11

**Comment by dwalkes** [comment]
Hi Everyone,
Our May monthly meeting will be held Thursday May 9th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/GjrANwHQd35a7K5n4ChrLarwzm7y1u

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9005%E2%80%9009

**Comment by dwalkes** [comment]
Hi Everyone,
Our June monthly meeting will be held Thursday June 13th at 9:30 AM MT.  See calendar link at 
https://teamup.com/event/show/id/uZkNbEMpn9SvQ1oH6FhLKKtkNsGTxy

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9006%E2%80%9013

**Comment by dwalkes** [comment]
Hi Everyone,
Our July monthly meeting will be held Thursday July 11th at 9:30 AM MT. See calendar link at
https://teamup.com/event/show/id/MjKHzE7r53HYQnvk4WFAM44JYy3k4B

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9007%E2%80%9011

**Comment by dwalkes** [comment]
Hi Everyone,
Our August monthly meeting will be held Thursday August 8th at 9:30 AM MT.  See calendar link at 
https://teamup.com/event/show/id/MYaiJdQHzpLbxABTy1jTFsr5uWRX6T

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9008%E2%80%9008

**Comment by dwalkes** [comment]
Hi Everyone,
Our September monthly meeting will be held Thursday September 12th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/juC6745fE4kCUfGgTm96dFpQvTanBi

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9009%E2%80%9012

**Comment by dwalkes** [comment]
Hi Everyone,
Our October monthly meeting will be held Thursday October 10th at 9:30 AM MT. See calendar link at https://teamup.com/event/show/id/DQwfTxaMYkeqwoAVa84iLWJTkTVFu2

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9010%E2%80%9010

**Comment by dwalkes** [comment]
Hi Everyone,
Our November monthly meeting will be held Thursday November 14th at 9:30 AM MT *Note the time shift due to DST ending in the US* 

See calendar link at https://teamup.com/event/show/id/sxxXpPCeJPSR95zAJ42EsUXD5JNtj9

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9011%E2%80%9014

**Comment by dwalkes** [comment]
Hi Everyone,
Our December monthly meeting will be held Thursday December 12th at 9:30 AM MT

See calendar link at https://teamup.com/event/show/id/Gk7bKAYphoQCytSJXat5XEWCj3kKAq

Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024%E2%80%9012%E2%80%9012

**Comment by dwalkes** [comment]
Hi Everyone,

Our January monthly meeting will be held Thursday January 9th at 9:30 AM MT

See calendar link at https://teamup.com/event/show/id/cKkKDraRY6Buty3WC1unNMDvAdNHCm

Hope you can attend!

**Reply by moto-timo** [reply]
Do we have a plan for February? March?

**Reply by dwalkes** [reply]
Yes, sorry, just updated with the Feb link. I believe you should be able to access the full calendar at https://teamup.com/ksfr2rp3ee3bzhq6q1 as well, it's just the 2nd Thursday of every month currently.

**Comment by dwalkes** [comment]
Hi Everyone,
Our February monthly meeting will be held Thursday February 13th at 9:30 AM MT
See calendar link at https://teamup.com/event/show/id/SYPXqN1nGB4EZGJQJUKDbivV6SqFg9
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9002%E2%80%9013

**Comment by dwalkes** [comment]
Hi Everyone,
Our March monthly meeting will be held Thursday March 13th at 9:30 AM MT
See calendar link at https://teamup.com/event/show/id/7JSPqz93t7G4ikUfysCSyX5xy5AvTF
Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9003%E2%80%9013

**Comment by dwalkes** [comment]
Hi Everyone,
Our April monthly meeting will be held Thursday April 10th at 9:30 AM MT
See calendar link at https://teamup.com/event/show/id/fCKpKXxaJHz36H5kEhvEbgYpnAEUs3
Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9004%E2%80%9010

**Comment by dwalkes** [comment]
Hi Everyone
Our May monthly meeting will be held **Friday** May 9th at 9:00 AM MT **Note the day and time change from previous schedule**

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/CQ7RSBGy4obsc8MTZD5vx1VGASmHKP

Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9005%E2%80%9009

**Comment by dwalkes** [comment]
Hi Everyone
Our June monthly meeting will be held Friday June 13th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/SVzJpDxayL4LUhPhdw5LkW61qCxvqu

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9006%E2%80%9013

**Comment by dwalkes** [comment]
Hi Everyone
Our July monthly meeting will be held Friday July 11th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/82ukFkDyTePGuDy63bdcxHFekfwpB8

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9007%E2%80%9011

**Comment by dwalkes** [comment]
Hi Everyone
Our August monthly meeting will be held Friday August 8th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/i4htoXmiSa22TGMsDkt7JDk2RMrFLx

Hope you can attend!

**Reply by dwalkes** [reply]
Meeting notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9008%E2%80%9008

**Comment by dwalkes** [comment]
Hi Everyone
Our September monthly meeting will be held Friday September 12th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/HtpXdx6TV7vhYPFpzxkaznV64zC3WD

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9009%E2%80%9012

**Comment by dwalkes** [comment]
Hi Everyone
Our October monthly meeting will be held Friday October 10th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/DuCG1CaCnLMsBbB96LF6bV4auRcjDD

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9010%E2%80%9010

**Comment by dwalkes** [comment]
Hi Everyone
Our November monthly meeting will be held Friday November 14th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/muoe378WugfCnFa8n6oJUHR2cLM4Rp

Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9011%E2%80%9014

**Comment by dwalkes** [comment]
Hi Everyone
Our December monthly meeting will be held Friday December 12th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/WwcXvdBh2EwuMvDR5JEGaHNQDaFLPD

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9012%E2%80%9012

**Comment by dwalkes** [comment]
Hi Everyone
Our January monthly meeting will be held Friday January 9th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/6LqUF1jtQjpAzQfVS1jHZ91x5Tbadb

Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2025%E2%80%9001%E2%80%9009

**Reply by madisongh** [reply]
I fixed the year for  that page: https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9001%E2%80%9009

**Comment by dwalkes** [comment]
Hi Everyone,
Our February monthly meeting will be held Friday February 13th at 9:00 AM Mt

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/4mQekca3sTXoJTGkKiWZtqqkuFUw8F
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9002%E2%80%9013 - I even got the year right this time!

**Comment by dwalkes** [comment]
Hi Everyone,
Our March monthly meeting will be held Friday March 13th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/yJzwzEDFpuyM7ngKqDJsomj6B92ypi
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9003%E2%80%9013

**Comment by dwalkes** [comment]
Hi Everyone,
Our April monthly meeting will be held Friday April 10th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/pLjtCJ1vMY7fBJoYW4Advkyzi91cam
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9004%E2%80%9010

**Comment by dwalkes** [comment]
Hi Everyone,
Our May monthly meeting will be held Friday May 8th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/3MaCqGfKiY63QVyKN1nZkVWnDkBCm1
Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9005%E2%80%9008

**Comment by dwalkes** [comment]
Hi Everyone,
Our June monthly meeting will be held Friday June 12th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/UL5V45XMEigaqWVeezU4H1JSedq8pU
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9006%E2%80%9012

**Comment by dwalkes** [comment]
Hi Everyone,
Our July monthly meeting will be held Friday July 10th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/KZJzX2grP6DDmn2r6r7H5FNV1thAhB
Hope you can attend!

**Reply by dwalkes** [reply]
Notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9007%E2%80%9009

**Comment by dwalkes** [comment]
Hi Everyone,
Our August monthly meeting will be held Friday August 14th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/pbf3iRkMXS7A8zfZ1ZCvTL2NV7u2Te
Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9008%E2%80%9014

**Comment by dwalkes** [comment]
Hi Everyone,
Our September monthly meeting will be held Friday September 11th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/gck13tcbDZ2W9iWZweCcPikEb7m5s9
Hope you can attend!

**Reply by dwalkes** [reply]
See notes at https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2026%E2%80%9009%E2%80%9011

**Comment by dwalkes** [comment]
Hi Everyone,
Our October monthly meeting will be held Friday October 9th at 9:00 AM MT

I've added a link to the calendar feed which should be updated and you can use to import into the calendar of your choice: https://ics.teamup.com/feed/ksfr2rp3ee3bzhq6q1/8794856.ics

See calendar link for the individual meeting at https://teamup.com/event/show/id/wfjcyqfLiMjiAf6LvazPYAMMoz6ors
Hope you can attend!
